completions/fio

```diff
@@ -70,12 +77,70 @@ _fio()
             _filedir -d
             return
             ;;
+        --ioengine)
+            _comp_cmd_fio__engines "$1"
+            COMPREPLY=($(compgen -W '${ret[@]}' -- "$cur"))
+            return
+            ;;
+        --exec_postrun | --exec_prerun)
+            COMPREPLY=($(compgen -c -- "$cur"))
+            return
+            ;;
+        --uid)
+            _uids
+            return
+            ;;
+        --gid)
+            _gids
+            return
+            ;;
+        --?*)
+            local IFS=$'\n'
+            local cmdhelp=($("$1" --cmdhelp=${prev#--} 2>/dev/null))
+            _comp_unlocal IFS
+            case ${cmdhelp[*]} in
```

Comment:
nounset in bash-4.2, 4.3.

```suggestion
            case ${cmdhelp[*]-} in
```

Reply:
Cool, I guess this will fix the CentOS 7 / Ubuntu 14 test failures I hadn't found time to track down.